static/quickstart.js

```diff
@@ -38,32 +30,8 @@
         log("Twilio.Device Error: " + error.message);
       });
 
-      device.on("connect", function (conn) {
-        log("Successfully established call!");
-        document.getElementById("button-call").style.display = "none";
-        document.getElementById("button-hangup").style.display = "inline";
-        volumeIndicators.style.display = 'block';
-        bindVolumeIndicators(conn);
-      });
-
-      device.on("disconnect", function (conn) {
-        log("Call ended.");
-        document.getElementById("button-call").style.display = "inline";
-        document.getElementById("button-hangup").style.display = "none";
-        volumeIndicators.style.display = 'none';
-      });
-
-      device.on("incoming", function (conn) {
-        log("Incoming connection from " + conn.parameters.From);
-        var archEnemyPhoneNumber = "+12093373517";
-
-        if (conn.parameters.From === archEnemyPhoneNumber) {
-          conn.reject();
-          log("It's your nemesis. Rejected call.");
-        } else {
-          // accept the incoming connection and start two-way audio
-          conn.accept();
-        }
+      device.on("incoming", function (call) {
+          incomingCallUI(call);
```

Comment:
Similar shortcut to the trick you are doing on line 61 😃 
```suggestion
      device.on("incoming", incomingCallUI);
```